src/app/containers/Video/index.jsx

```diff
@@ -12,15 +14,23 @@ import { filterForBlockType } from '../../helpers/blockHandlers';
 const VideoContainer = ({ blocks }) => {
   const captionBlock = filterForBlockType(blocks, 'caption');
   const aresMediaBlock = filterForBlockType(blocks, 'aresMedia');
+  const metadata = videoMetadata(aresMediaBlock);
   if (!aresMediaBlock) {
     return null;
   }
 
   return (
-    <Figure>
-      <Video />
-      {captionBlock ? <Caption block={captionBlock} video /> : null}
-    </Figure>
+    <>
+      {metadata ? (
+        <Helmet>
+          {<script type="application/ld+json">{metadata}</script>}
```

Comment:
The contents of this script tag need to be JSON.stringified, since Helmet expects that.
The way to check the output is working is by running Storybook locally and going to the [Video Container story](http://localhost:9001/?selectedKind=Video%20Container&selectedStory=with%20a%20caption) 
This is the error that is currently visible: 
<img width="1402" alt="Screen Shot 2019-05-13 at 10 49 51" src="https://user-images.githubusercontent.com/3028997/57612537-ecb95f00-756c-11e9-8e1e-9b5b9c7d0906.png">

Reply:
Sorry about that. Fixed.